Dockerfile

```
@@ -1,2 +1,11 @@
 FROM golang:1.17.0-alpine3.14
 
+ENV PATH /usr/local/git/bin:$PATH
+
+RUN apk add --no-cache git
+
+WORKDIR PATH
```

```suggestion
WORKDIR $PATH
```

環境変数を使う場合は、`$PATH`もしくは`${PATH}`というように記載しないといけないはずです。